backend/config/config.go

```diff
@@ -438,3 +438,20 @@ func migrateUserLanguage(appconf *AppConfig) {
 		delete(frontconf, "userLanguage")
 	}
 }
+
+// migrateActiveTokens removes tokens from AccountsConfig.
+func migrateActiveTokens(accountsConf *AccountsConfig) {
+	for _, account := range accountsConf.Accounts {
+		// copy account so it does not take a reference to the loop scoped account.
+		ac := account
```

Comment:
that should not be needed here right? the account var is not accessed after the loop iteration.

Reply:
True, I thought accessing an attribute on it could lead to problems but that would just be the case if it was in a go routine.